ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -91,7 +91,8 @@
       .put("/submit/sslreports/", "1585144") //
       .build();
 
-  private static final String REDACTED_SEARCH_CODE_VALUE = "other.scrubbed";
+  private static final String DESKTOP_REDACTED_SEARCH_CODE_VALUE = "other.scrubbed";
+  private static final String MOBILE_REDACTED_SEARCH_CODE_VALUE = "scrubbed";
```

Comment:
Since `other.scrubbed` is already deployed for desktop, I don't want to change it. For mobile, `.` is used as a separator, so I don't want to include a `.` in the scrubbed value, so I think these need to be different (but as similar as possible).

Reply:
My summary of discussion in Slack:

> Sounds like mconnor’s preference out of the available options would be other-scrubbed. That value avoids confusion over code vs. channel and is unique enough to be unlikely to conflict with a future legitimate value. I am going to move forward with using other-scrubbed for pipeline sanitization when the code isn’t in the allowlist

This got a thumbs-up from mconnor.